---a/net/minecraft/util/CombatEntry.java
+++b/net/minecraft/util/CombatEntry.java
@@ -2,8 +2,8 @@
 
 import net.minecraft.entity.EntityLivingBase;
 
-public class CombatEntry
-{
+public class CombatEntry {
+
     private final DamageSource damageSrc;
     private final int field_94567_b;
     private final float field_94568_c;
@@ -12,8 +12,7 @@
     private final float field_94564_f;
     private static final String __OBFID = "CL_00001519";
 
-    public CombatEntry(DamageSource damageSrcIn, int p_i1564_2_, float p_i1564_3_, float p_i1564_4_, String p_i1564_5_, float p_i1564_6_)
-    {
+    public CombatEntry(DamageSource damageSrcIn, int p_i1564_2_, float p_i1564_3_, float p_i1564_4_, String p_i1564_5_, float p_i1564_6_) {
         this.damageSrc = damageSrcIn;
         this.field_94567_b = p_i1564_2_;
         this.field_94568_c = p_i1564_4_;
@@ -22,34 +22,27 @@
         this.field_94564_f = p_i1564_6_;
     }
 
-    public DamageSource getDamageSrc()
-    {
+    public DamageSource getDamageSrc() {
         return this.damageSrc;
     }
 
-    public float func_94563_c()
-    {
+    public float func_94563_c() {
         return this.field_94568_c;
     }
 
-    public boolean func_94559_f()
-    {
+    public boolean func_94559_f() {
         return this.damageSrc.getEntity() instanceof EntityLivingBase;
     }
 
-    public String func_94562_g()
-    {
+    public String func_94562_g() {
         return this.field_94566_e;
     }
 
-    public IChatComponent func_151522_h()
-    {
+    public IChatComponent func_151522_h() {
         return this.getDamageSrc().getEntity() == null ? null : this.getDamageSrc().getEntity().getDisplayName();
     }
 
-    public float func_94561_i()
-    {
+    public float func_94561_i() {
         return this.damageSrc == DamageSource.outOfWorld ? Float.MAX_VALUE : this.field_94564_f;
     }
-}
+}
